webserver/webserver/src/main/java/io/helidon/webserver/ServerConfiguration.java

```diff
@@ -278,13 +310,17 @@ static Builder builder(Config config) {
         private final Map<String, SocketConfiguration> sockets = new HashMap<>();
         private int workers;
         private Tracer tracer;
+        private Duration maxShutdownTimeout;
+        private Duration shutdownQuietPeriod;
         private ExperimentalConfiguration experimental;
         private Optional<Transport> transport;
         private Context context;
         private boolean printFeatureDetails;
 
         private Builder() {
             transport = Optional.ofNullable(null);
```

Comment:
I know this is not part of this PR, but there is `Optional.empty()` that can be used (for future reference)